data/data/bootstrap/bootstrap-in-place/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
How is the output of this script being captured in the `gather bootstrap` log bundle?

Reply:
Good catch, should be added.